src/main/java/com/example/wbdvsp21finalprojectyeswr21serverjava/services/UserService.java

```diff
@@ -0,0 +1,42 @@
+package com.example.wbdvsp21finalprojectyeswr21serverjava.services;
+
+import com.example.wbdvsp21finalprojectyeswr21serverjava.models.User;
+
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.servlet.http.HttpSession;
+
+@Service
+public class UserService {
+    List<User> users = new ArrayList<User>();
+
+    public User register(User user, HttpSession session) {
```

Comment:
Thanks for restructuring the java server code. I would expect you to implement the login/register/profile UI which can take user input and store info in the UserService. Currently we only use url to make sure the service is functional.

Reply:
UI was implemented in our React Repo.